x-pack/lib/monitoring/inputs/metrics.rb

```diff
@@ -187,5 +209,24 @@ def fetch_global_stats
         }
       }
     end
+
+    def extract_cluster_uuids(stats)
+      result = stats.extract_metrics([:stats, :pipelines, :main, :config], :cluster_uuids)
+      if result && !result[:cluster_uuids].empty?
+        cluster_uuids = result[:cluster_uuids]
+        @logger.info("Found cluster_uuid from elasticsearch output plugins, ignoring setting monitoring.cluster_uuid. Using cluster_uuids: ", :cluster_uuids => cluster_uuids)
```

Comment:
I still have mixed feelings about mentioning ignoring the `cluster_uuid` setting when it has not been set. As long as we document the setting appropriately, the `cluster_uuid` setting shouldn't be of concern to most users and we should probably only draw attention to it when it is being used - maybe with an info message if it is being used, and a warning message if it is overridden by the values drawn from the plugins.

Reply:
I agree with you, citing something that the user hasn't configured is not good, reworded with a7e9c3868444b47999048a5e10ca00d569849ce7